backend/src/connectors/authorisation/base.ts

```diff
@@ -143,16 +144,22 @@ export class BasicAuthorisationConnector {
   ): Promise<Array<Response>> {
     const entities = await authentication.getEntities(user)
 
-    // Does the user hold a role on the model the parent is on?
-    const hasModelRole = (await authentication.getUserModelRoles(user, model)).length !== 0
-
     return Promise.all(
       accessRequests.map(async (request) => {
         // Does any individual in the access request share an entity with our user?
         const isNamed = request.metadata.overview.entities.some((value) => entities.includes(value))
 
-        // If they are not listed on the model
-        if (!isNamed && !hasModelRole && [AccessRequestAction.Delete, AccessRequestAction.Update].includes(action)) {
+        /**
+         * Reject if:
+         *  - user is not named on the access request
+         *  - user is not the owner of the model
+         *  - the user is trying to delete or update an existing AR
+         */
+        if (
+          !isNamed &&
+          (await missingRequiredRole(user, model, ['owner'])) &&
```

Comment:
I think this should be either of 'owner' or 'contributor'.  I would expect someone with the permissions to upload a model to also download it.

Reply:
Download a model? This is for the permissions when trying to update or delete an access request?